modules/core/src/passes/pick-layers-pass.ts

```diff
@@ -124,7 +117,7 @@ export default class PickLayersPass extends LayersPass {
   }
 
   protected shouldDrawLayer(layer: Layer): boolean {
-    return layer.props.pickable && layer.props.operation === OPERATION.DRAW;
+    return layer.isPickable();
```

Comment:
Rename this `layer.shouldDrawWhenPicking()`? `props.pickable` & `layer.isPickable()` have diverged